<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdatePluginDialogue</class>
 <widget class="QDialog" name="UpdatePluginDialogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>通过 Git 更新所有 Plugin</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="gitPathLabel">
         <property name="text">
          <string>Git仓库路径</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="gitLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="BtnAddGit">
       <property name="text">
        <string>添加仓库路径到列表中</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="GitPathListView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="BtnUpdate">
       <property name="text">
        <string>更新/下载插件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnClose">
       <property name="text">
        <string>关闭窗口</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
